package.json

```diff
@@ -134,5 +134,8 @@
   "eslintConfig": {
     "extends": "react-app"
   },
-  "dependencies": {}
+  "dependencies": {
```

Comment:
These should be in devDependencies.

Make sure to use `yarn add <package> --dev` or `npm install <package> --save-dev`